app/policies/proposal_policy.rb

```diff
@@ -20,8 +20,7 @@ def can_show!
   end
 
   def can_create!
-    # TODO restrict by client_slug
-    true
+    slug_matches? || @user.admin?
```

Comment:
I'm still new to this area, so please forgive what may be a stupid question:
When would this be false? And do we have a test for that situation?